backend/hct_mis_api/config/fragments/drf_spectacular.py

```diff
@@ -0,0 +1,9 @@
+SPECTACULAR_SETTINGS = {
```

Comment:
I think we can combine this file with a drf.py

Reply:
this is seoperate liblrary